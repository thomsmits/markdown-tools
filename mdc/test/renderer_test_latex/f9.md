# Chapter
## Slide

A __`#codeterm`__
