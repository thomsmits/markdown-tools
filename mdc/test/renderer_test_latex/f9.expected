# Chapter

## Slide

A \lstinline[literate={-}{{\textminus}}1{-\}{{\textminus}\}2,language=Java,style=inline]|#codeterm|\margincodeterm{\#codeterm}\vspace{0.1mm}
